syntax = "proto3";

package proto;

option go_package= "github.com/dangquyitt/shadowify/video-service/gen";

service TranscriptionService {
    rpc GetTranscript (GetTranscriptRequest) returns (GetTranscriptResponse);
}

message GetTranscriptRequest {
  string video_url = 1;
}

message GetTranscriptResponse {
  string transcript = 1;
}